vars/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 # Scratch directory to use for downloading the installation script
 nix_script_dir: "/tmp"
+support_flakes: false
```

Comment:
How about naming this just `nix_flakes`?

Also, this should be defined in `defaults/main.yml` (which I guess must be created), not `vars/main.yml`. Otherwise the variable will have [higher precedence](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#understanding-variable-precedence) than play vars, host_vars, group_vars, etc.